src/client/components/formComponent/formComponent.js

```diff
@@ -1,32 +1,42 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import RenderFormFields from '../renderFormFields/renderFormFields';
```

Comment:
If we use everywhere index.js for path's, please, add here too, and don't call two times "renderFormFields"
👨‍💻🙏

Reply:
you all obsessed with this index, I don't))

Reply:
I think you all should use the same structure
Use index.js everywhere or don’t everywhere